WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -563,9 +579,14 @@
         session[:locale] = :fr
       end
 
-      it 'creates an announcement post' do
+      it 'creates an announcement post and expects organizers to receive a notification email' do
         competition.update_attributes(start_date: "2011-12-04", end_date: "2011-12-05")
-        get :post_announcement, params: { id: competition }
+        organizer = FactoryBot.create :user
+        competition.organizers << organizer
+        expect(CompetitionsMailer).to receive(:notify_organizer_of_announced_competition).and_call_original
```

Comment:
I intentionally left the `with` clause. Same reasoning as for the announcement email preview.